### DISCUSSION TITLE
11.14.2022 Agenda

### DISCUSSION THREAD

**Discussion by TCEmkenn** [Third-Coast/website#59] [General]
- Competition Updates (15 minutes)
- EOY Campaign (15 minutes)
- Future-facing website (30 minutes)

@brendanmetzger @mayags

**Comment by brendanmetzger** [comment]
mail delivery and 'envelope appeal'. Visual interest is a given, but start thinking of things like '

**Comment by brendanmetzger** [comment]
canonical urls for things for 'entry guide',  'offerings', etc. instead of plastic urls (ie., permalinks) so they don't have annual import and possible duplication of similar content

**Comment by brendanmetzger** [comment]
https://mcachicago.org/about/who-we-are/artists
